File: python/day21.py
hour = int(input())
minute = int(input())
minAngle = 6*minutes
hourAngle = 30 * hour
angle = abs(minAngle-hourAngle)
if angle == 0:
    print("Both are at same point")
else:
    print("angle between hour and minutes is = ",angle)

File: cpp/day21.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    int hour,minute;
    cin>>hour>>minute;
    double minAngle = 6 * minute;
    double hourAngle = 30.0 * hour;
    double angle = abs(minAngle-hourAngle);
    if(angle == 0.0)
        cout<<"both are at same point";
    else
        cout<<"angle between minutes and hour is = "<<angle;
}
